web/templates/layout/_segment.html.eex

```diff
@@ -0,0 +1,91 @@
+<script type="text/javascript">
```

Comment:
The file does some interpolation to inject the `SEGMENT_WRITE_KEY`, I tried creating a `.js` file with the `.eex` extension but it didn't seem to be working. I think it's probably a configuration issue, maybe there's a better way?